lib/rubocop/cop/rails/time_zone.rb

```diff
@@ -96,8 +100,19 @@ def safe_method(method_name, node)
           end
         end
 
+        def check_localtime(node)
+          p = node
+          while !p.nil? && p.send_type?
+            break if extract_method(p) == :localtime
+            p = p.parent
```

Comment:
Ok, I've renamed `p` to `node` in `extract_method_chain` and in this case add `selector_node`.